bpfman-operator/config/bpfman-operator-deployment/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: quay.io/bpfman/bpfman-operator
   newName: quay.io/bpfman/bpfman-operator
-  newTag: latest
+  newTag: v0.4.1
```

Comment:
Was this on purpose?

Reply:
Nope good catch, maintaining these manifests are kind of a PITA :)